Echo.json5

```diff
@@ -212,8 +212,8 @@
       name: 'StripeStagingPublishableKey',
       content: '${StripeStagingPublishableKey}',
     },
-    { name: 'KillSwitchBuildMinimum', content: '8.5.0' },
-    { name: 'KillSwitchBuildMinimumAndroid', content: '8.5.0' },
+    { name: 'KillSwitchBuildMinimum', content: '8.35.0' },
+    { name: 'KillSwitchBuildMinimumAndroid', content: '8.35.0' },
```

Comment:
Hey @brainbicycle did I do this right? Your investigation showed this:

```
~/C/eigen (brian/track-failed-deeplinks|✔) $ git which-release a84f4a7                                                                                                                                                                                                   (base) 
ios-8.35.0-2024.03.20.12-submission
android-8.35.0-1674645638-submission
```

So I wasn't sure if this should be `8.35.0` or `8.36.0`. 🤔

Reply:
should be 8.36 actually!

Reply:
Ok updated!